packages/framework/azure-service-utils/package.json

```diff
@@ -36,7 +36,8 @@
   },
   "dependencies": {
     "@fluidframework/protocol-definitions": "^0.1027.1000",
-    "jsrsasign": "^10.2.0"
+    "jsrsasign": "^10.2.0",
+    "uuid": "^8.3.1"
```

Comment:
This dependency was completely missing. I think the only reason it has been building OK is because of lerna dependency hoisting.